_includes/components/breadcrumbs.njk

```diff
@@ -0,0 +1,13 @@
+<div class="breadcrumbs-block">
+	<div class="container">
+		<div class="govuk-breadcrumbs">
+			<ol class="govuk-breadcrumbs__list">
+				<li class="govuk-breadcrumbs__list-item"><a class="govuk-breadcrumbs__link" href="{{ '/' | url }}">Home</a></li>
+				{% if ( breadcrumbs_parent ) %}
+				<li class="govuk-breadcrumbs__list-item"><a class="govuk-breadcrumbs__link" href="{{ breadcrumbs_parent_url }}">{{ breadcrumbs_parent }}</a></li>
```

Comment:
Right now this only support three breadcrumbs, we need four levels on the individual team/board profile pages https://github.com/11ty/eleventy-navigation

Reply:
@SamMed1 It sounds like individual profile pages are out of scope for the next deadline. So I'm happy if we open a new ticket to re-factor this later

Reply:
Thanks for clarifying.